conf/initd/webvirtmgr-novnc-redhat

```diff
@@ -32,6 +37,7 @@ start() {
     [ -x $exec ] || exit 5
 
     echo -n $"Starting $prog: "
+    export PYTHONPATH=$PYTHONPATH:/usr/lib/python2.6/site-packages/Django-1.5.6-py2.6.egg
```

Comment:
Where this Django package is coming from? (pip, rpm?), because in https://github.com/retspen/webvirtmgr/blob/master/requirements.txt version is 1.5.5; be careful that you are coupling this script to a specific version of Django, in a specific path.